docs/howto/create-an-uberjar/index.adoc

```diff
@@ -34,9 +37,3 @@ $ mvn package
 $ java -jar ./target/myapp-thorntail.jar
 ----
 
-ifndef::product[]
-.Related Information
-
-* xref:creating-a-hollow-jar[]
```

Comment:
In this PR, sometimes "related information" are kept, sometimes they are removed. Why's that?

Reply:
Again, I believe some of the _Related information_ sections didn't add value, so I got rid of them. Some of them are in files that I will modify in a later PR, too.

Reply:
Ah OK.